packages/superset-ui-control-utils/src/types.ts

```diff
@@ -52,13 +52,8 @@ export interface DatasourceMeta {
 
 export interface ControlPanelState {
   form_data: QueryFormData;
-  datasource?: DatasourceMeta | null;
-  options?: ColumnMeta[];
```

Comment:
What happen to `options`?

Reply:
It's not actually a valid property for `ControlPanelState`, judging by the code in `superset-frontend` (I did `console.log` on when `mapStateToProps` is called to debug). Removing it doesn't break any current type checking so I assumed it was good.

Reply:
got it.